gulp/tasks/deploy.js

```diff
@@ -93,6 +95,23 @@ gulp.task('deploy:dist:files', function() {
       .pipe(tsFilter.restore);
 });
 
+//
+// Generates JS bundles and puts them in the dist/ folder.
+//
+// In addition to the UMD bundle coming from the build task, the following
+// formats are generared: CJS, ESM, ES6.
+// See "rollup.config.js" for more informations.
```

Comment:
informations => information (there is no plural form of information, same for data)